build.zig

```diff
@@ -218,7 +218,6 @@ pub fn build(b: *Build) !void {
                     .argv = &.{
                         "git",
                         "rev-parse",
-                        "--short",
```

Comment:
this will cause the output in bun install to change

and also a few other places where we mention the git sha

we should have two of these variables and one the full sha and the other the partial sha

Reply:
Looking at the usages of `git_sha`, it's actually expected to be the full SHA. `bun install` truncates the variable, but other usags like `Bun.revision` use it as-is.